pkgs/by-name/pu/pulsemeeter/package.nix

```
@@ -0,0 +1,63 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  libappindicator,
+  libpulseaudio,
+  gobject-introspection,
+  wrapGAppsHook4,
+  ladspaPlugins,
+# noise-suppression-for-voice,
+# pulse-vumeter,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pulsemeeter";
+  version = "1.2.14";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "theRealCarneiro";
+    repo = "pulsemeeter";
+    tag = "v${version}";
+    hash = "sha256-QTXVE5WvunsjLS8I1rgX34BW1mT1UY+cRxURwXiQp5A=";
+  };
+
+  build-system = [
+    python3.pkgs.setuptools
+  ];
+
+  nativeBuildInputs = [
+    gobject-introspection
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    libappindicator
```

```suggestion
    bash
    libappindicator
```
`pmctl` is a bash script. At the moment it runs via `#!/usr/bin/env bash`. Put bash here to use nixpkgs' bash instead.